src/publish/deep-freeze.js

```diff
@@ -0,0 +1,16 @@
+exports.default = function deepFreeze(object) {
```

Comment:
Do we really need deepFreeze? I mean it's harmless but if we're gonna roll our own helper, maybe we can do without it for simplicity's sake?

Reply:
Agree. Tbh none of the tests really need to rely on it anyway.